content/applications/marketing/sms_marketing/essentials/sms_essentials.rst

```diff
@@ -176,6 +175,41 @@ is visualized as different graphs and charts, which can be sorted and grouped in
    :align: center
    :alt: Reporting page in SMS Marketing.
 
+.. tip::
+   SMS messages can be sent using automated actions in Odoo. Odoo *Studio* is required to use
+   automated actions. To install it, go to :menuselection:`Apps --> Search...` bar and search for
+   `studio`. If it is not already installed, click :guilabel:`Install`. Adding the *Studio*
+   application upgrades the subscription status to *Custom*, which increases the cost. Consult
+   `support <https://www.odoo.com/contactus>`_ or reach out to the database's customers success
+   manager with any questions on upgrading.
+
+   To use automated actions, navigate in :ref:`developer mode <developer-mode>`, to
+   :menuselection:`Settings app --> Technical menu --> Automation section --> Automated Actions`.
+   Then, click :guilabel:`New` to create a new action.
+
+   Enter an :guilabel:`Action Name` and select a :guilabel:`Model` to implement this action on. Set
+   the :guilabel:`Trigger` to either :guilabel:`On Creation`, :guilabel:`On Update`, :guilabel:`On
+   Creation & Update`, :guilabel:`On Deletion`, :guilabel:`Based on Form Modification`, or
+   :guilabel:`Based on Timed Condition`.
+
+   Based on the selection for the :guilabel:`Trigger`, additional fields may populate.
+
+   Under the :guilabel:`Apply on` field a record filter using a domain can be created. Ensure a
+   model has been selected before setting any domains on the :guilabel:`Apply on` field. Click
+   :guilabel:`Edit Domain` to set record parameters.
+
+   Under :guilabel:`Action To Do` drop-down field, select :guilabel:`Send SMS Text Message`. Next,
+   set the :guilabel:`SMS Template`, and choose whether the SMS message should be logged as a note,
+   by ticking the box next to :guilabel:`Log as Note`.
+
+   .. image:: sms_essentials/automated-action-sms.png
+      :align: center
+      :alt: Automated action template with action to do, SMS template and log as note highlighted.
+
+   Ensure the :guilabel:`Active` toggle is set to *on* for the automated action to run.
```

Comment:
might be nice to place this between the 'Model' and 'Trigger' instructions just so the reader can follow along with how items/fields appear on the form, sequentially